website/layouts/index.html

```diff
@@ -55,7 +55,7 @@ <h3 class="of-heading of-heading--lg">Go</h3>
                     <li>Use the SDK CLI to build and generate the operator deployment manifests</li>
                   </ol>
            
-                  <a href="#" class="of-button of-button--tertiary">
+                  <a href="https://sdk.operatorframework.io/docs/golang/quickstart/" class="of-button of-button--tertiary">
```

Comment:
Does it work to drop the domain portion so it works on all domains/builds? Ditto for the other absolute links with `sdk.operatorframework.io`.
```suggestion 
                  <a href="/docs/golang/quickstart/" class="of-button of-button--tertiary">
```

This is non-blocking so we don't have to wait for CI again.